packages/shared/src/lib/func.ts

```diff
@@ -72,3 +72,17 @@ export const wrapStopPropagation =
     event.stopPropagation();
     callback();
   };
+
+export const sortAlphabeticallyByProperty =
```

Comment:
I would argue that for a generic function it doesn't need to actually care if it has a property or not, it would just expect a and b as 2 strings.

Reply:
Yes but we almost always order by property and if I had not created it like that you would have to do `array.sort((a, b) => sortAlphabetically(a.name, b.name))` vs now you can do `array.sort(sortAlphabeticallyByProperty('name'))` which I think is more convenient. 

Additionally in the first one you are calling 2 functions in each iteration: closure and then `sortAlphabetically` vs in second example you only call one function each iteration that gets created by `sortAlphabeticallyByProperty`.

Reply:
We can add additional one that is flat if needed in the future when needed.

Reply:
Makes sense for the usage, we can always create a separate func then if needed for plain strings.